src/components/Card/Chart/PeriodAdCountCard.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+
+import asyncGetAdCountList from "../../../api/keyword/asyncGetAdCountList";
+import { PERIOD_TYPE } from "../../../config/constants";
+import PeriodToggleButton from "../../Button/PeriodToggleButton";
+import StackBarChart from "../../Chart/StackBarChart";
+import PeriodPagination from "../../Pagination/PeriodPagination";
+import { keepPreviousData, useQuery } from "@tanstack/react-query";
+import PropTypes from "prop-types";
+
+const PeriodAdCountCard = ({ keywordId }) => {
+  const [cursorId, setCursorId] = useState("");
+  const [period, setPeriod] = useState(PERIOD_TYPE.WEEKLY);
+  const hasKeywordId = !!keywordId;
+
+  const {
+    data: chartData,
+    isPlaceholderData,
+    isError,
+  } = useQuery({
+    queryKey: ["adCount", keywordId, period, cursorId],
+    queryFn: () => asyncGetAdCountList(keywordId, cursorId, period),
+    enabled: hasKeywordId,
+    placeholderData: keepPreviousData,
+  });
+
+  if (chartData === undefined) {
+    return null;
+  }
+
+  if (isError || chartData?.message?.includes("Error occured")) {
+    return (
+      <div className="flex flex-col gap-6 w-1/2 h-full p-10 border-2 rounded-md justify-center items-center">
+        게시물 광고 여부 차트를 불러오는 데 실패했습니다.
+      </div>
+    );
+  }
+
+  return (
+    <article className="flex flex-col gap-6 w-1/2 h-full p-10 border-1 rounded-md">
+      <div className="flex-shrink-0 bg-green-100/20 px-10 py-5 rounded-[2px]">
+        <span className="text-18 font-semibold">게시물 광고 여부</span>
+        <PeriodToggleButton
+          keywordId={keywordId}
+          period={period}
+          setPeriod={setPeriod}
+          setCursorId={setCursorId}
+        />
+      </div>
```

Comment:
각 카드마다 `title`과 `toggleButton`의 세로 정렬이 안되어 있는 것 같아요. 

```suggestion
      <div className="flex justify-between items-center flex-shrink-0 bg-green-100/20 px-10 py-5 rounded-[2px]">
        <span className="flex items-center text-18 font-semibold">게시물 광고 여부</span>
        <PeriodToggleButton
          keywordId={keywordId}
          period={period}
          setPeriod={setPeriod}
          setCursorId={setCursorId}
        />
      </div>
```

Reply:
와! 오늘의 게시물 수 카드를 제외하곤 계속 정렬하려고 시도했지만 안됐는데, 제안주신 코드로 하니 바로 정렬되네요..! 👍